contrib/review-trailers-tooling/NVIM_RUNTIME_PATH/plugin/insert-github-user-info.vim

```diff
@@ -0,0 +1,14 @@
+" Replace word under cursor (which should be a GitHub username)
+" with some user info ("Full Name <email@address>").
+" If info cout not be found, "Not found" is inserted.
+function! <SID>InsertGitHubUserInfo()
```

Comment:
Nice! why not export the function, so it easily can be remapped? Also slight issue, it doesn't work if username is prefixed with `@`